### DISCUSSION TITLE
Override/delete `label` in custom Combo

### DISCUSSION THREAD

**Discussion by sklinov** [antvis/G6#3452] [Q&A]
Is there a way to override `label` in custom combo create with `G6.registerCombo`? 
The docs say that 
> Do not add text shape for label in drawShape, it will be added and updated automatically by the base class;` 

I'd like to keep the prop in the config, but want to render it in a different way.
Or how can I remove the label? I'll be using a custom `text` + `image` shapes instead

**Comment by Cherry860** [comment]
Maybe don't inherit rect or circle, and customize them completely.

G6.registerCombo ('comboName', options, expendedComboName)  without expendedComboName
